src/main/kotlin/com/terraformation/backend/customer/db/OrganizationStore.kt

```diff
@@ -594,6 +607,17 @@ class OrganizationStore(
           .and(ORGANIZATION_USERS.USER_ID.eq(userId))
           .fetchOne(ORGANIZATION_USERS.ROLE_ID)
 
+  private fun getUserEmail(userId: UserId): String =
+      dslContext.select(USERS.EMAIL).from(USERS).where(USERS.ID.eq(userId)).fetchOne(USERS.EMAIL)
+          ?: throw UserNotFoundException(userId)
+
+  private fun validateTerraformationContactEmail(userId: UserId) =
+      getUserEmail(userId).let { it ->
+        if (!it.endsWith(suffix = "@terraformation.com", ignoreCase = false)) {
```

Comment:
Do we check for malformed emails anywhere? For instance, `foo@bar@terraformation.com` would pass the checks but isn't a valid email.

Reply:
That wouldn't end up being a valid user, so it'd fail the user ID lookup later on; I think this is fine.

However, I'd probably do `ignoreCase=true` here because `TERRAFORMATION.COM` is technically a valid spelling of our email domain.

Reply:
Oops, thanks for catching that, intended it to be `ignoreCase=true` but kept thinking `caseSensitive` and used false. Will fix this.